### PR TITLE
Getting All Customers

### PR DESCRIPTION
**Pull request by danielKNS**
Getting All customers

### CODE REVIEW

File: java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Customer/CustomerController.java [hunk 1]
```diff
@@ -18,4 +21,10 @@ public void createNewCustomer(){
         customerService.createNewCustomer();
     }
 
+    @GetMapping("/customersList")
+    public List<Customer> allCustomers(){
+        System.out.println("Getting ALL Customers");
+        return customerService.gettingAllCustomers();
+    }
+
```
Comment (lgc13) [inline java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Customer/CustomerController.java]: the url path is not Restful. This should just be `/customers`

Comment (lgc13) [reply java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Customer/CustomerController.java]: also, sysout or log?

File: java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Customer/CustomerService.java [hunk 2]
```diff
@@ -29,4 +31,11 @@ public void createNewCustomer(){
             System.out.println("A new row has been inserted!!!");
         }
     }
+// -----------GETTING ALL CUSTOMER (DATABASE)------------ //
+    public List<Customer> gettingAllCustomers(){
+        String sql = "SELECT * FROM customer";
+        //query means searching for something
+        List<Customer> customersList = jdbcTemplate.query(sql, new BeanPropertyRowMapper<>(Customer.class));
+        return  customersList;
+    }
```
Comment (lgc13) [inline java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Customer/CustomerService.java:71]: do we need these comments?

**Review by lgc13 — APPROVED** [review]

File: java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Customer/CustomerController.java (hunk 1 above)
Comment (danielKNS) [reply java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Customer/CustomerController.java]: Done.

File: java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Customer/CustomerService.java (hunk 2 above)
Comment (danielKNS) [reply java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Customer/CustomerService.java:71]: these comments helps me to read my code. So i am going to stay with them.

**Review by lgc13 — APPROVED** [review]

**Pull request merged**